src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookEndpoint.java

```diff
@@ -81,7 +81,7 @@ private String getEventKey(StaplerRequest request) {
         return eventKey;
     }
 
-    private <T> T parse(StaplerRequest request, Class<T> type) throws ServletException {
+    private <T> T parse(StaplerRequest request, Class<T> type) {
```

Comment:
I can't comment on the line, but you can now also remove the `throws ServletException` on the `doTrigger` method above